crypto-ffi/src/generic/context/mod.rs

```diff
@@ -414,7 +413,7 @@ impl CoreCryptoContext {
         &self,
         conversation_id: Vec<u8>,
         key_packages: Vec<Vec<u8>>,
-    ) -> CoreCryptoResult<MemberAddedMessages> {
+    ) -> CoreCryptoResult<Option<Vec<String>>> {
```

Comment:
Does it make sense to introduce a typedef or something to indicate what the list of string that's optionally returned is?